src/utils/misc.ts

```diff
@@ -85,3 +85,5 @@ export function removeItemFromArray<T>(array: T[], index: number) {
   newArray.splice(index, 1)
   return newArray
 }
+
+export const cleanUrl = (url: string) => url.replace('https://', '')
```

Comment:
Nit: maybe the name could be a bit more explicit? Like `removeUrlScheme`? Or `removeUrlHttps` :P 

<img width="726" alt="image" src="https://github.com/alephium/desktop-wallet/assets/6715943/05eda735-365a-4b58-bffd-5efde83434e7">

Reply:
I guess we could make the name more explicit, but I like that the current name is small. No objection though.

Reply:
I'm just expecting a more profound cleaning when I see a name like that :D But no objection either.